tools/pipelines/templates/upload-json-steps.yml

```diff
@@ -16,7 +16,7 @@ parameters:
 
 # Determines if artifact should be published as latest-v*.tar.gz (for release branches)
 - name: uploadAsLatestRelease
-  type: boolean
+  type: string
```

Comment:
Can we make this a string enum that accepts "true" or "false"? That makes it clearer what the valid options are.